README.md

```diff
@@ -109,6 +112,16 @@ Anybody with a preview link (including not logged-in users) will be able to view
 2. Via one-time usage. If the "One-time use" checkbox is selected, the preview URL will only work a single time, and then the token will become invalid. If this box is not selected, a preview URL can be used any number of times before expiration.
 3. Via post status changes. If a post moves out of VIP Workflow's set of extended post statuses, tokens will no longer be valid. For example, a post moved to "Publish" will no longer have valid preview tokens.
 
+#### Editorial Metadata
+
+VIP Workflow adds a "Editorial Metadata" section to the post sidebar, which allows for additional data to be included with the post such as "Needs Legal Review". This can be managed under the plugin's settings, to get a visual for all of the configured editorial metadata fields.
```

Comment:
Can we add a screenshot of the sidebar's Editorial Metadata section? It looks good.

Reply:
Yip, I mentioned in the PR that I'd add that after the PR was finalized just to ensure only the final design gets uploaded to the media branch

Reply:
My bad, forgot about that! 👍